opencj/weapons.gsc

```diff
@@ -203,4 +204,21 @@ _onSettingSlowReload(value)
     {
         self setClientCvar("perk_weapReloadMultiplier", 0.5);
     }
+}
+
+_onSettingSmallCrosshair(value)
+{
+    if(value)
+    {
+        self SetSpreadOverride(1);
+    }
+    else
+    {
+        self ResetSpreadOverride();
+    }
+}
+
+setWeaponSpread() 
+{
+    _onSettingSmallCrosshair(self openCJ\settings::getSetting("smallcrosshair"));
```

Comment:
Why this? Now the _onSettingSmallCrosshair will get called twice: when setting was loaded, and upon player spawn?

Reply:
Well yes, just don't pass an update function